### DISCUSSION TITLE
Connection attempt to node 'clouseau@127.0.0.1' aborted since it cannot handle ["BIG_CREATION"].

### DISCUSSION THREAD

**Discussion by yhilem** [apache/couchdb#4728] [Q&A] [answered]
Hi,
I set flag enableSearch to true to deploy couchb with helm charts  (https://github.com/apache/couchdb-helm/tree/main/couchdb).
Two containers (couchdb and clouseau) are deployed in the POD.

But the in the container couchdb i have these traces :
`[error] 2023-08-13T17:54:24.783422Z couchdb@couchdbstd-couchdb-0.couchdbstd-.....svc.cluster.local <0.3445.2> -------- ** 'couchdb@couchdbstd-couchdb-0.couchdbstd-couchdb.....svc.cluster.local': Connection attempt to node 'clouseau@127.0.0.1' aborted since it cannot handle ["BIG_CREATION"].**`

And in the container clouseau i have these traces :
```
2023-08-13 17:56:24,790 scalang.node.ServerHandshakeHandler [ERROR] Channel closed during handshake
2023-08-13 17:56:24,791 scalang.node.ErlangHandler [DEBUG] handler message HandshakeFailed('couchdb@couchdbstd-couchdb-0.couchdbstd-couchdb......svc.cluster.local)
2023-08-13 17:56:24,796 scalang.node.ErlangHandler [INFO] channel disconnected org.jboss.netty.channel.DefaultChannelPipeline$DefaultChannelHandlerContext@6bf8e30e [id: 0x1438c12c, /127.0.0.1:50410 :> /127.0.0.1:33189] DISCONNECTED. peer: null
2023-08-13 17:56:24,797 scalang.node.ServerHandshakeHandler [ERROR] Channel closed during handshake
```

I have found at (https://erlangforums.com/t/keep-getting-connection-attempt-from-node-aaaaaa-aaaaaaa-rejected-since-it-cannot-handle-big-creation/2544/2) this explanation 
```
Distribution flag DFLAG_BIG_CREATION became mandatory from OTP23. It says: This flag is mandatory. If not present, the connection is refused. You’re probably connecting node running OTP<23 and node running OTP>=23.
```
But don't know what config args are missed ?
Thanks for your help.

**Comment by rnewson** [comment]
Hi, you'll need the latest version of clouseau to handle the updated erlang RPC protocol.

**Comment by yhilem** [comment, marked as answer]
I use :
searchImage:
  repository: kocolosk/couchdb-search
  tag: 0.2.0
  pullPolicy: IfNotPresent

With clouseau version is [2.17.0](https://github.com/cloudant-labs/clouseau/tree/2.17.0)

The latest clouseau version is https://github.com/cloudant-labs/clouseau/releases/tag/2.21.6

Thank you, I will build this latest image.

**Comment by yhilem** [comment]
Hi
Thank you for your help.
I tested with clouseau latest version and it's ok :
```
>curl -k -u xxx:yyy https://........io/test01/_design/search/_search/posts?q=ea885d7d-7af2-4858-b7bf-6fd01bcd4544
{"total_rows":0,"bookmark":"g2o","rows":[]}
```
